scripts/application_startup/customization_dev.groovy

```diff
@@ -335,7 +335,9 @@ void update_omegat_prefs(localPrefsPath) {
 
     // Try to guess the property type, Boolean, Integer or String
     remotePrefs.each { prop ->
-        if (prop.value == "true" || prop.value == "false") {
+        if (prop.value instanceof File) {
```

Comment:
```
        if (prop.value instanceof File) {
```

Do you know where a File object is set as a Preference? I can't find any reference.

In any case, it would probably be better to do the following (to catch other cases)

```java
        } else {
            Preferences.setPreference(prop.key, prop.value.toString())
        }
```

Reply:
```Groovy
local_scripts_dpath = new File(config_dir.toString() + File.separator + "scripts")
```

and later inside update_omegat_prefs()
```Groovy
    remotePrefs.scripts_dir = local_scripts_dpath
```

In any case, it might be better indeed to fix it right there, by changing it into the following:
```Groovy
    remotePrefs.scripts_dir = local_scripts_dpath.toString()
```


Speaking of which; what's the purpose of forcing integer looking strings to be converted to Integer, does it make a difference somewhere? If not then your suggestion is indeed better.

Reply:
> Speaking of which; what's the purpose of forcing integer looking strings to be converted to Integer, does it make a difference somewhere? If not then your suggestion is indeed better.

Inside the code of OmegaT, few preferences are directly set as Integer/Boolean (and not parsed from a String like most other prefs). If we try to set these preferences with a String, it will throw a ClassCastException